GVFS/GVFS.Hooks/Program.cs

```diff
@@ -109,7 +109,7 @@ private static void RemindUpgradeAvailable()
             // sure that the reminder is displayed only 10% of the times a git 
             // command is run, check that the random number is between 0 and 10, 
             // which will have a probability of 10/100 == 10%.
-            int reminderFrequency = 10;
+            int reminderFrequency = 100;

```

Comment:
This is only to product reminder messaging during all the git command tests. Will remove before final commit.